modules/util/source-formatter/src/main/resources/sourcechecks.xml

```diff
@@ -1608,6 +1608,11 @@
 			<description name="Checks if the property value `-includeresource` or `Include-Resource` exists and removes it" />
 			<weight>1</weight>
 		</check>
+		<check name="UpgradeCommerceOrderConstantsCheck">
+			<category name="Upgrade" />
+			<description name="Replace departed constants of CommerceOrderConstants by CommerceOrderPaymentConstants equivalents" />
```

Comment:
@lazhcsilva, sorry, was a misspelling of my part, should be "Replace deprecated constants of CommerceOrderConstants by CommerceOrderPaymentConstants equivalents"